plugins/modules/proxmox_kvm.py

```diff
@@ -482,6 +482,24 @@
       - Timeout for operations.
     type: int
     default: 30
+  tpmstate0:
+    description:
+      - A hash/dictionary of options for the Trusted Platform Module disk.
+      - A TPM state disk is required for Windows 11 installations.
+    suboptions:
+      storage:
+        description:
+          - O(tmpstate0.storage) is the storage identifier where to create the disk.
```

Comment:
`tmp` and `tpm` are right next to each other on the keyboard... sorry for that ;)
```suggestion
          - O(tpmstate0.storage) is the storage identifier where to create the disk.
```

Reply:
hah, I hadn't even noticed :p